iOS/Flows/Home/_New/new_ConversationViewController.swift

```diff
@@ -46,7 +49,8 @@ class new_ConversationViewController: FullScreenViewController, CollectionViewIn
 
     init(conversation: Conversation?) {
         if let conversation = conversation {
-            self.conversationController = ChatClient.shared.channelController(for: conversation.cid)
+            self.conversationController
+            = ChatClient.shared.channelController(for: conversation.cid, messageOrdering: .bottomToTop)
```

Comment:
Message ordering allows us to load the messages right to left. Oldest first, newest last.

Reply:
perfecto.